src/components/button.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+function Button({clickHandler, type, children}){
+    return(
+            <button onClick={clickHandler}
+                    type={type}
+            >
+                {children}
+            </button>
+    );
+}
```

Comment:
Dit is nu een soort doorgeefluik. Je had net zo goed direct het button element kunnen gebruiken